skin.confluence/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="skin.confluence" version="4.6.3" name="Confluence" provider-name="Jezz_X, Team Kodi">
+<addon id="skin.confluence" version="4.6.5" name="Confluence" provider-name="Jezz_X, Team Kodi">
```

Comment:
remove the website and email tags

Reply:
fix also the forum url

Reply:
Please drop the comment block in https://github.com/xbmc/repo-skins/blob/d48a1128847d44878a9940dba956d27239a89e55/skin.confluence/language/resource.language.en_gb/strings.po#L19 although this is more of an addon-checker issue

Reply:
The addon-checker is correct, `#` has to be followed by a space (` `), dot (`.`), colon (`:`), comma (`,`) or vertical bar (`|`).